listener_clients/M5AtomMatrix-listener/tallyarbiter-m5atom/tallyarbiter-m5atom.ino

```diff
@@ -320,6 +331,7 @@ void evaluateMode() {
     actualColor.replace("#", "");
     String hexstring = actualColor;
     long colorNumber = (long) strtol( &hexstring[1], NULL, 16);
+ // This order is to compensate for Matrix needing grb.
```

Comment:
oh! I see.. but it didn't seem to change anything here.  Using TallyArbiter 3.0, I needed to change the colors in the Settings for the different Busses, as those are the ones that seem to be used.

Reply:
Hi @abourget ... We've been running multiple Matrices in production for several weeks now and this code seems to work well for us. Let's see if we can figure out what's different with your setup. Starting with the server - are you using the default color assignments for Program and Preview? Second question would be does the web interface on the server reflect the standard color assignments? That is, red for Program and green for Preview?